apps/web/frontend/components/weekGrid/weekGridTaskRow.tsx

```diff
@@ -36,43 +42,135 @@ const WeekGridTaskRowFragment = graphql(`
   }
 `)
 
+interface WorkHourProps {
+  date: string
+  isLocked: boolean
+  user: {
+    id: string
+    name?: string | null
+    image?: string | null
+  }
+  workHour?: {
+    duration: number
+  } | null
+}
+
 interface WeekGridTaskRowProps {
   task: FragmentType<typeof WeekGridTaskRowFragment>
   isDataOutdated?: boolean
+  userIds: string[]
 }
 
-export const WeekGridTaskRow = ({ task: taskFragment, isDataOutdated = false }: WeekGridTaskRowProps) => {
+export const WeekGridTaskRow = ({ task: taskFragment, isDataOutdated = false, userIds }: WeekGridTaskRowProps) => {
   const task = useFragment(WeekGridTaskRowFragment, taskFragment)
-  const taskDurations = task.workHourOfDays
-    .map((workHour) => workHour.workHour?.duration ?? 0)
-    .reduce((previous, current) => previous + current, 0)
+  const daysOfWeek = [1, 2, 3, 4, 5, 6, 0] // Monday to Sunday
```

Comment:
Let's put it above the `WeekGridTaskRow` component maybe
```suggestion
  const DAYS_OF_WEEK = [1, 2, 3, 4, 5, 6, 0]
```